cli/tests/unit/read_text_file_test.ts

```diff
@@ -93,7 +93,7 @@ Deno.test(
     queueMicrotask(() => ac.abort());
     const error = await assertRejects(
       async () => {
-        await Deno.readFile("cli/tests/testdata/assets/fixture.json", {
+        await Deno.readTextFile("cli/tests/testdata/assets/fixture.json", {
```

Comment:
Why change it?

Reply:
this is the readTextFile test file. seems whoever wrote the test originally made a mistake